lms/templates/wiki/base.html

```diff
@@ -6,6 +6,8 @@
 {% block bodyclass %}view-incourse view-wiki{% endblock %}
 
 {% block headextra %}
+  <script type="text/javascript" src="/i18n.js"></script>
+  <script src="{% static 'js/vendor/backbone-min.js' %}"></script>
```

Comment:
We need backbone here because the timer bar will still display if the user navigates to other areas of the course outside of the courseware tab. At the time I implemented this, backbone.js was not available on the page, I can check again since you mentioned that there's been some recent work regarding the JS pipeline.